meta-mender-core/recipes-mender/mender-client/mender-client.inc

```diff
@@ -44,7 +44,6 @@ FILES_${PN} += "\
     ${datadir}/mender/modules/v3/rpm \
     ${datadir}/mender/modules/v3/script \
     ${datadir}/mender/modules/v3/single-file \
-    ${sysconfdir}/mender.conf \
```

Comment:
Question:
Why not fix the path?

Reply:
If it is not needed, why explicitly add it to `FILES`?

Reply:
Hmm, odd. I thought this was part of the QA check, and that we always install the `/etc/mender/mender.conf` (local) conf file. I was expecting this to fail if :thinking:

Reply:
You are right in that if a file is installed and nos listed in `FILES`, bitbake QA checks will fail.

But some paths are added to `FILES` by inherited classes and I suspect that the full `/etc/mender` dir is added this way.